src/Main.java

```diff
@@ -1,5 +1,17 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        int[] numbers = new int[]{123, 432, 1, 7, 23, 5, 43};
```

Comment:
You should use two different sets of values to test your sorting algorithms. Bubble sort has its own set of numbers, and insertion sort has its own.